projects/NXP/devices/iMX6/linux/linux.arm.conf

```diff
@@ -134,10 +142,10 @@ CONFIG_RD_LZMA=y
 CONFIG_RD_XZ=y
 CONFIG_RD_LZO=y
 CONFIG_RD_LZ4=y
+CONFIG_INITRAMFS_COMPRESSION=".gz"
```

Comment:
is this right?

Reply:
In the sense that this option wasn't present at all, then yes it's now using the default - the original config is/was wrong (if you run `make oldconfig` on master for iMX6 then `gz` will pop in). What compression algorithm do you want to use?

Reply:
Same with Dragonboard, the `CONFIG_INITRAMFS_COMPRESSION` option is missing there too.

Reply:
I point it out because as far as I know we haven't been compressing the initramfs, only the kernel as a whole (at least for everything but aarch64)

Reply:
That's mostly right, in which case Dragonboard and iMX6 should have been using `CONFIG_INITRAMFS_COMPRESSION=""` which is in line with RPi and Generic. Allwinner seems to be doing it's own thing.

From master (note the absence of Dragonboard and iMX6):
```
$ git grep INITRAMFS_COMPRESSION
projects/Allwinner/linux/linux.aarch64.conf:CONFIG_INITRAMFS_COMPRESSION=".lz4"
projects/Allwinner/linux/linux.arm.conf:CONFIG_INITRAMFS_COMPRESSION_NONE=y
projects/Allwinner/linux/linux.arm.conf:# CONFIG_INITRAMFS_COMPRESSION_GZIP is not set
projects/Allwinner/linux/linux.arm.conf:CONFIG_INITRAMFS_COMPRESSION=""
projects/Generic/linux/linux.x86_64.conf:CONFIG_INITRAMFS_COMPRESSION=""
projects/RPi/devices/RPi/linux/linux.arm.conf:CONFIG_INITRAMFS_COMPRESSION=""
projects/RPi/devices/RPi2/linux/linux.arm.conf:CONFIG_INITRAMFS_COMPRESSION=""
projects/RPi/devices/RPi4/linux/linux.arm.conf:CONFIG_INITRAMFS_COMPRESSION=""
```

I'll update Dragonboard and iMX6 to use `CONFIG_INITRAMFS_COMPRESSION=""`

Reply:
Strange but thanks for taking corrective action.

It's complicated because I think it would be good to use initramfs compression on aarch64 kernels that use `Image` and don't have kernel compression.

arm kernels can be compressed using the zImage so that is fine to leave the initramfs uncompressed.

Opinions?

Reply:
That sounds logical.

I can limit the `CONFIG_INITRAMFS_COMPRESSION=""` change to `arm` if you prefer? Removing initramfs compression affects multiple options as you can see: http://ix.io/1OGy

Reply:
Extra iMX6 commit now added.